.github/workflows/e2e-ui-tests.yml

```diff
@@ -21,52 +21,52 @@ jobs:
           - 5432:5432
 
     steps:
-     - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
 
-     - name: Cache node modules
-       uses: actions/cache@v2
-       env:
-         cache-name: cache-node-modules
-       with:
-         # npm cache files are stored in `~/.npm` on Linux/macOS
-         path: ~/.npm
-         key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
-         restore-keys: |
-           ${{ runner.os }}-build-${{ env.cache-name }}-
-           ${{ runner.os }}-build-
-           ${{ runner.os }}-
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          # npm cache files are stored in `~/.npm` on Linux/macOS
+          path: ~/.npm
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
 
-     - name: Setup Node.js
-       uses: actions/setup-node@v1
-       with:
-        node-version: '14.x'
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "14.x"
 
-     - name: Install project dependencies
-       run: yarn install --frozen-lockfile
+      - name: Install project dependencies
+        run: yarn install --frozen-lockfile
 
-     - name: Copy .env-ci to .env
-       run: cp apps/backend/test/.env-ci apps/backend/.env
+      - name: Copy .env-ci to .env
+        run: cp apps/backend/test/.env-ci apps/backend/.env
 
-     - name: Create/migrate db
-       run: |
-        yarn backend sequelize-cli db:create
-        yarn backend sequelize-cli db:migrate
-        yarn backend sequelize-cli db:seed:all
+      - name: Create/migrate db
+        run: |
+          yarn backend sequelize-cli db:create
+          yarn backend sequelize-cli db:migrate
+          yarn backend sequelize-cli db:seed:all
 
-     - name: Build Heimdall
-       run: yarn build
-       env:
-        NODE_ENV: production
+      - name: Build Heimdall
+        run: yarn build
+        env:
+          NODE_ENV: production
 
-     - name: Cypress run
-       uses: cypress-io/github-action@v2
-       with:
-         start: yarn start
-         wait-on: http://127.0.0.1:3000
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          start: yarn start, yarn run cypress-test mock-oauth
```

Comment:
Can you just say `yarn run start cypress-test mock-oauth` here?

Reply:
Tried this locally, doesn't seem to start the mock server